app/form/field/PercentageField.js

```diff
@@ -0,0 +1,10 @@
+
+Ext.define('CpsiMapview.form.field.PercentageField',
+    {
+        extend: 'CpsiMapview.form.field.NumericField',
+        xtype: 'cmv_percentagefield',
+        symbol: '%',
+        decimalPrecision: 2,
```

Comment:
Additional comma and blank line can be removed

Reply:
done